### PR TITLE
Implement rubocop

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -50,13 +43,13 @@ task 'test:spec' => 'test:spec:run'
...
-task :test => 'test:spec'
-task :default => :test
+task test: 'test:spec'
+task default: :test
 
 begin
   require 'rubygems'
   require 'github_changelog_generator/task'
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError
```
Comment (rwaffen) [inline Rakefile:52]: add a comment otherwise rubocop maybe nagging about empty rescue

**Review by arjenz — APPROVED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -10,6 +10,18 @@ env:
...
 
 jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Ruby ${{ matrix.ruby }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.1"
```
Comment (rwaffen) [inline .github/workflows/test.yml:20]: why only 3.1 here?

**Review by rwaffen — COMMENTED** [review]
LGTM, but see comments

File: Rakefile (hunk 1 above)
Comment (bastelfreak) [reply Rakefile:52]: odd that rubocop didn't already complain. I will have a look.

File: .github/workflows/test.yml (hunk 2 above)
Comment (bastelfreak) [reply .github/workflows/test.yml:20]: we have a dependency to a vmware lib, which doesn't support ruby 3.2 yet :(

**Pull request merged**